build/scripts/bump-image/main.go

```diff
@@ -0,0 +1,106 @@
+// Copyright 2023 Google LLC All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package main implements a program to increment the new tag for the given examples image. Run this script using `make bump-image IMAGENAME=<imageName> VERSION=<version>`
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+var (
+	imageName       string
+	version         string
+	versionPattern  *regexp.Regexp
+	targetedFolders = map[string]bool{
+		"build":    true,
+		"examples": true,
+		"install":  true,
+		"pkg":      true,
+		"site":     true,
+		"test":     true,
+	}
+)
+
+func init() {
+	flag.StringVar(&imageName, "imageName", "", "Image name to update")
+	flag.StringVar(&version, "version", "", "Version to update to")
```

Comment:
If I understood correctly, the `version` here is the version we are going to update from, instead of update to, right?

Reply:
This flag specifies the version that we want to update to the new tag. However, I am open to changing its description to either `version to update from` or simply `version to update` if that makes more sense.